third_party/move/move-compiler-v2/tests/more-v1/parser/invalid_call_lhs_parens_around_name.exp

```diff
@@ -1,6 +1,12 @@
 
 Diagnostics:
-error: Calls to function values other than inline function parameters not yet supported
+error: function values outside of inline functions not supported in this language version
+  ┌─ tests/more-v1/parser/invalid_call_lhs_parens_around_name.move:3:10
+  │
+3 │         (foo)()
+  │          ^^^
+
+error: function values outside of inline functions not supported in this language version
```

Comment:
nit: duplicated error from the user perspective, but I guess it is because one is for referring to the function via the `(function_name)` syntax, the other is for the actual call to such a function.

Reply:
Also these are temporary as we upgrade to the new version eventually